tools/test-harnesses/src/tests/findByLgId.spec.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from 'react';
```

Comment:
could findBy & getBy be merged into one test file? (since they're defined in the same source file)

Reply:
I like the idea of separating the tests in case we decide to export more of the testid queries in the future.

Reply:
I feel like tripling the number of test files (find, get, query) might get a bit overwhelming. I'd either have one `byLgId.spec` file, or sort these 3 files into a `byLgId/` directory

Reply:
I like the idea of the `byLgId` directory -- i'll update